lib/list_validator.rb

```diff
@@ -1,11 +1,18 @@
 class ListValidator
+  def self.call(constraint: {}, test: [])
+    test.map!(&:to_sym)
+    new(constraint.keys).all_valid?(test)
+  end
```

Comment:
Do we have any tests for this behaviour?

Reply:
(Nit: would it possible to condense this class into a single function?)

Reply:
As it's a refactor I believe the current tests should still be valid.

I did wonder if it would be easier to just add a method to the calculator as it's only used in one, but thought that pehaps that was a change too far.

Reply:
I've added unit test for the `.call` method